### PR TITLE
Remove unnecessary else statements

### PR DESCRIPTION
**Pull request by dullgiulio**
Whenever it makes the code easier to follow, use early return to
avoid else statements.

It's a dirty job, but someone has to do it.

### CODE REVIEW

File: db/password.go [hunk 1]
```diff
@@ -49,13 +49,12 @@ func (r *passwordInfoRepo) Create(tx repo.Transaction, pw user.PasswordInfo) (er
...
 	_, err = r.get(tx, pw.UserID)
-	if err != nil {
-		if err != user.ErrorNotFound {
-			return err
-		}
-	} else {
+	if err == nil {
 		return user.ErrorDuplicateID
 	}
+	if err != nil && err != user.ErrorNotFound {
```
Comment (arnehormann) [inline db/password.go]: here and some other places below: `err != nil` can be deleted, it's always true because of the `if err == nil` above

Comment (dullgiulio) [inline db/password.go]: True, correct this and the others.

**Comment by bobbyrullo** [comment]
Thanks @dullgiulio ! Did you use some automated tool for this, or did you just dig in and read a ton of code?

Minor nit: our commit messages start with subsystem; in your case it touches almost everywhere so it should be like:

>*: Remove unnecessary else branches

**Comment by bobbyrullo** [comment]
otherwise, LGT

**Comment by dullgiulio** [comment]
I used grep and it wasn't that much code to check ;)

**Comment by bobbyrullo** [comment]
Congrats on having the first merged non-CoreOS contribution!

**Pull request merged**